### PR TITLE
Updated AnyMovie url

### LINKED ISSUE #2565: AllMoviesForYou extension is showing error when it tried to play any video
### Source information

AllMoviesForYou 13.12

### Source language

English

### Steps to reproduce

Just open the extension & try to play anything.

### Expected behavior

The extension should've played the video when I clicked on it.

### Actual behavior

When I tried to play any video, it's showing
"Video list is empty" error message 

Screenshot_20231128-151552972

### Aniyomi version

App version: 0.14.6-6697 (standard, 2b8000b81, 106, 2023-11-14T22:48Z) Android version: 12 (SDK 31) Android build ID: SKQ1.210821.001.WW_Phone-31.0210.0210.303-0 release-keys Device brand: asus Device manufacturer: asus Device name: ASUS_I003_1 Device model: ASUS_I003DD Device product name: WW_I003D

### Android version

Android 12

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.3.10](https://github.com/aniyomiorg/aniyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://aniyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/aniyomiorg/aniyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by kaoru2**
Fixes #2565 
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

**Comment by Secozzi** [comment]
Don't change the package name if you're just updating the url

File: src/en/anymovie/src/eu/kanade/tachiyomi/animeextension/en/anymovie/AnyMovie.kt [hunk 1]
```diff
@@ -22,11 +22,11 @@ import org.jsoup.nodes.Element
...
 import uy.kohesive.injekt.api.get
 
-class AllMovies : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
+class AnyMovie : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
 
-    override val name = "AllMoviesForYou"
+    override val name = "AnyMovie"
 
-    override val baseUrl = "https://allmoviesforyou.net"
+    override val baseUrl = "https://anymovie.cc/"
```
Comment (Secozzi) [inline src/en/anymovie/src/eu/kanade/tachiyomi/animeextension/en/anymovie/AnyMovie.kt:29]:
```suggestion
    override val baseUrl = "https://anymovie.cc"
```
Remove trailing slash

File: src/en/anymovie/src/eu/kanade/tachiyomi/animeextension/en/anymovie/AnyMovie.kt [hunk 2]
```diff
@@ -22,11 +22,11 @@ import org.jsoup.nodes.Element
 import uy.kohesive.injekt.Injekt
 import uy.kohesive.injekt.api.get
 
-class AllMovies : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
+class AnyMovie : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
 
-    override val name = "AllMoviesForYou"
+    override val name = "AnyMovie"
```
Comment (Secozzi) [inline src/en/anymovie/src/eu/kanade/tachiyomi/animeextension/en/anymovie/AnyMovie.kt:27]: You should keep the old `id` if you're changing the name

**Comment by kaoru2** [comment]
> Don't change the package name if you're just updating the url

The name changed from AllMoviesForYou to AnyMovie and I did something like in AniWave, should I not?

**Comment by Secozzi** [comment]
> > Don't change the package name if you're just updating the url
> 
> The name changed from AllMoviesForYou to AnyMovie and I did something like in AniWave, should I not?

You shouldn't change the package name, as you can see on aniwave, the package name uses "nineanime" not "aniwave"

**Comment by jmir1** [comment]
You need to override the extension id.

Read this: https://github.com/aniyomiorg/aniyomi-extensions/blob/master/CONTRIBUTING.md#renaming-existing-sources

File: src/en/allmovies/src/eu/kanade/tachiyomi/animeextension/en/allmovies/AnyMovie.kt
Comment (jmir1) [inline src/en/allmovies/src/eu/kanade/tachiyomi/animeextension/en/allmovies/AnyMovie.kt:1]: You need to override `id`

**Review by jmir1 — CHANGES_REQUESTED** [review]

**Comment by fakebhai** [comment]
you mentioned it `fixes #2565` , so is it still WIP? because last time I checked changing baseurl doesn't fix it

**Pull request closed without merging**